examples/gatekeeper-no-root/README.md

```diff
@@ -0,0 +1,112 @@
+# gatekeeper: No Root
+
+### Overview
+
+This example demonstrates how to declaratively run the [gatekeeper]
+function to validate resources using gatekeeper constraints.
+
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/gatekeeper-no-root
+```
+
+There are 3 resources: a `ConstraintTemplate`, a `NoRoot` and a `Deployment`.
```

Comment:
I think using `DisallowRoot` wording instead of `NoRoot` might reflect the usage better, WDYT ?